src/main/java/seedu/address/model/person/PersonEvent.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.model.person;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class PersonEvent {
+    private final int date;
+    private final int month;
+    private final Person person;
+    private final String description;
+
+    /**
+     * Constructs a {@code PersonEvent}
+     *
+     * @param localDate Date of the event.
+     * @param person Person whom the event belongs to.
+     * @param description A description of the event.
+     */
+    public PersonEvent(LocalDate localDate, Person person, String description) {
+        this.date = localDate.getDayOfMonth();
+        this.month = localDate.getMonthValue();
+        this.person = person;
+        this.description = description;
+    }
+
+    public int getDate() {
+        return date;
+    }
+
+    public int getMonth() {
+        return month;
+    }
+
+    public Person getPerson() {
+        return person;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public static String getBirthdayDescription(Person person) {
```

Comment:
`PersonEvent` should have no knowledge of `Birthday` and `Event` classes, put these two helper methods in their respective methods. They could be named `toUi()` too.

Reply:
I'm unable to put them in their respective classes because it requires the Person's name, which is not stored on the `Birthday` and `Event` class.

Reply:
Ok, got it. Lets leave it here first